test/arrays/intents/in/no-domain-linkage.chpl

```diff
@@ -0,0 +1,11 @@
+
```

Comment:
I'm having trouble seeing what this test is trying to test? It seems to work on master...

Reply:
It does work on master, but I didn't find an example of this behavior.

It is verifying that 'in A : []' works like variable initialization in the sense that 'A' shares the same domain as the actual.